### PR TITLE
Add xvfb-run package, update gtk-doc, gtkmm and glib, fix glibmm tests

### PR DESCRIPTION
**Pull request by ericonr**

### CODE REVIEW

**Comment by ericonr** [comment]
glib tests complete on glibc but fail on musl. Investigating, though it's likely not urgent.

File: srcpkgs/cabbage/template [hunk 1]
```diff
@@ -49,6 +49,7 @@ Projucer() {
 	# https://github.com/WeAreROLI/JUCE/issues/422
 	# The HOME variable is spoofed because it looks for headers in
 	# ~/SDKs/ (path hardcoded)
+	# TODO: use xfbv-run script for testing
```
Comment (fosslinux) [inline srcpkgs/cabbage/template:52]: typo

**Review by fosslinux — CHANGES_REQUESTED** [review]

File: srcpkgs/cabbage/template (hunk 1 above)
Comment (ericonr) [reply srcpkgs/cabbage/template]: Thanks!

Comment (ericonr) [reply srcpkgs/cabbage/template]: Thanks!

**Comment by ericonr** [comment]
Added `fwupd` fix up to use the new gtk-doc.

**Pull request merged**